CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### v1.7.3 March 08, 2023
+
+* Upgrade IAP SDK for android `1.6.1` and for iOS `1.6.1`.
```

Comment:
Nit: Please capitalize Android and ios -> iOS.
Thanks!

Reply:
done